cmd/evm/main.go

```diff
@@ -175,7 +175,7 @@ func NewEnv(state *state.StateDB, transactor common.Address, value *big.Int) *VM
 		state:      state,
 		transactor: &transactor,
 		value:      value,
-		time:       uint64(time.Now().Unix()),
+		time:       new(big.Int).SetInt64(time.Now().Unix()),
```

Comment:
You could use `big.NewInt(...)` instead of creating an int and then setting its value.